src/pages/MatchEdition/EditMatch.tsx

```diff
@@ -0,0 +1,135 @@
+import { motion } from 'framer-motion'
+import dayjs from 'dayjs'
+import { useForm } from 'react-hook-form'
+import { Box, Stack } from '@mui/material'
+import { useParams } from 'react-router-dom'
+import CircularProgress from '@mui/material/CircularProgress'
+import { useEffect, useState } from 'react'
+import pick from 'lodash.pick'
+import { Match as MatchType } from '../../types/Match'
+import { useMatch, useMatchUpdate } from '../../queries/match'
+
+import { MatchParticipants } from '../../components/Match/MatchParticipants'
+import { MatchDetails } from '../../components/Match/MatchDetails'
+
+import { animationProps } from '../../styles/animation'
+import { FabSubmit } from '../../components/FabSubmit'
+import { getErrorMessage } from '../../utils/api'
+import { Alert } from '../../components/Alert'
+
+export const MatchEdition = () => {
+  const { id } = useParams()
+  const matchId = id ? parseInt(id) : 0
+  const [isReady, setIsReady] = useState(false)
+
+  const { data, isLoading: isLoadingMatch, isError, error } = useMatch(matchId)
+  const { mutate: mutateMatchDetails } = useMatchUpdate()
+
+  const { handleSubmit, control, reset } = useForm<MatchType>({
+    defaultValues: {
+      boardgameName: '',
+      startedAt: '',
+      endedAt: '',
+      notes: '',
+      participants: [],
+    },
+  })
+
+  useEffect(() => {
+    if (data) {
+      reset(
+        pick(data, [
+          'boardgameName',
+          'startedAt',
+          'endedAt',
+          'notes',
+          'participants',
+        ])
+      )
+    }
+
+    if (isLoadingMatch === false) {
```

Comment:
Maybe add a comment here to describe why we can't use `isLoadingMatch` directly...